doc/develop/env_vars.rst

```diff
@@ -187,6 +187,8 @@ the :ref:`toolchain <gs_toolchain>` used to build Zephyr applications.
   ``ZEPHYR_TOOLCHAIN_VARIANT=llvm``, use :envvar:`LLVM_TOOLCHAIN_PATH`. (Note
   the capitalization when forming the environment variable name.)
 
+You might need to update some of these variables when you :ref:`update Zephyr SDK toolchain <gs_toolchain_update>`.
```

Comment:
```suggestion
You might need to update some of these variables when you :ref:`update the Zephyr SDK toolchain <gs_toolchain_update>`.
```

Similar issue throughout -- shouldn't we say "the" Zephyr SDK toolchain, with a definite article?